app/templates/orders/new.hbs

```diff
@@ -0,0 +1,27 @@
+<div class="ui two column container stackable doubling left grid" >
```

Comment:
@CosmicCoder96 Thanks for the suggestion! I didn't notice it! I have updated the code. Please review it now.